homeassistant/components/unifiprotect/event.py

```diff
@@ -35,6 +35,26 @@ class ProtectEventEntityDescription(ProtectEventMixin, EventEntityDescription):
         ufp_event_obj="last_ring_event",
         event_types=[EventType.RING],
     ),
+    ProtectEventEntityDescription(
+        key="doorbell_nfc",
+        translation_key="doorbell_nfc",
+        name="Doorbell-NFC",
```

Comment:
not 100% sure, but I think we can drop the `name` since it has a translation key as long as the translations are up to date 

`python3 -m script.translations develop --all`

Reply:
![image](https://github.com/user-attachments/assets/2bee7a3a-0293-4dec-986a-5bd2da10d9bd)
I'm not that deep into the topic, but that's the name of the event that I see in the gui

Reply:
We should be able to drop the names and use the translation_keys instead 
like
https://github.com/home-assistant/core/blob/efa86293aa47f8f7b5395cd0b48f72ea8002b674/homeassistant/components/nest/strings.json#L90

Reply:
https://github.com/home-assistant/core/blob/efa86293aa47f8f7b5395cd0b48f72ea8002b674/homeassistant/components/nest/event.py#L45